run_test.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+cd testsuite/gnat2goto
+! ./testsuite.py -j 2 2>&1 >/dev/null | tee /dev/tty | grep -q -m 1 "ERROR" && ../../gnat2goto/install/bin/unit_tests
```

Comment:
I'd recommend running the integration testsuite and the unit tests independently from each other

Reply:
So we want to run all the tests and fail if there was at least one error? Or just run the two sets of tests separately and stop each one with the first error?